pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/maler/example/LetterExample.kt

```diff
@@ -110,6 +130,20 @@ object LetterExample : AutobrevTemplate<LetterExampleDto> {
                 }
             }
 
+            paragraph {
+                showIf(FeatureToggles.pl7231ForventetSvartid.expr().enabled() and true.expr()) {
```

Comment:
Hva er grunnen til `and true.expr()` her?

Reply:
Litt vilkårleg, men mest for å få brukt `and` på toggle-`showIf`, for å verifisere at det funkar.